catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/utils/TableOpsUtils.java

```diff
@@ -46,4 +153,39 @@ private static void checkColumnAutoIncrement(String fieldName, boolean autoIncre
         String.format(
             "Paimon does not support auto increment column. Illegal column: %s.", fieldName));
   }
+
+  private static void checkNestedColumn(String[] fieldNames) {
+    Preconditions.checkArgument(
+        fieldNames.length == 1,
+        String.format(
+            "Paimon does not support update nested column. Illegal column: %s.",
+            fieldName(fieldNames)));
+  }
+
+  public static String[] fieldName(String fieldName) {
+    return new String[] {fieldName};
+  }
+
+  public static String fieldName(String[] fieldName) {
+    return DOT.join(fieldName);
+  }
+
+  private static String fieldName(ColumnChange columnChange) {
+    return fieldName(columnChange.fieldName());
+  }
+
+  private static Move move(String fieldName, ColumnPosition columnPosition)
+      throws UnsupportedOperationException {
+    if (columnPosition instanceof After) {
+      return Move.after(fieldName, ((After) columnPosition).getColumn());
+    } else if (columnPosition instanceof Default) {
+      return null;
```

Comment:
does this acttly works? seems `testUpdateColumnPosition` not cover this case.

Reply:
1. for `AddColumn` in Paimon: https://github.com/apache/paimon/blob/master/paimon-core/src/main/java/org/apache/paimon/schema/SchemaManager.java#L245-L254
if the Move is null, then the new column will be added  to the end.

2. `Default` position is not supported for `UpdateColumnPosition` in Paimon

Reply:
> does this acttly works

https://github.com/apache/gravitino/pull/4349/files#diff-6bea46f4cb01e05fd11727d1fddd956728bcb163d08f4be08d39ee7eeb729b9fR532-R533
there is a test case about `AddColumn` with `Default` position.

Reply:
Iceberg did some logic when updateColumnPosition is `default`, but this seems ridiculous after another thought, tracked in #4412